package.json

```diff
@@ -1,22 +1,22 @@
 {
   "name": "@itwin/insights-client",
-  "version": "0.2.1",
+  "version": "0.3.0",
   "description": "Insights client for the iTwin platform",
   "main": "lib/cjs/insights-client.js",
   "module": "lib/esm/insights-client.js",
   "typings": "lib/cjs/insights-client",
   "scripts": {
-    "build": "npm run -s build:cjs && npm run -s build:esm",
+    "build": "npm run -s build:cjs && npm run -s build:esm && npm run copy",
     "build:cjs": "tsc 1>&2 --outDir lib/cjs",
     "build:esm": "tsc 1>&2 --module es6 --outDir lib/esm",
+    "copy": "copyfiles -u 4 src/integration-test/utils/imodels-client-test-utils/assets/* src/integration-test/utils/imodels-client-test-utils/assets/changesets/* lib/cjs/integration-test/utils/imodels-client-test-utils",
     "lint": "eslint \"*/**/*.{js,ts}\"",
     "lint:fix": "npm run lint -- --fix",
     "lint:copyright": "node scripts/copyright-linter.js",
-    "webpackTests": "webpack --config ./src/test/utils/webpack.config.js 1>&2",
-    "test": "npm run build && npm run -s webpackTests && npm run -s test:chrome",
-    "test:chrome": "certa -r chrome",
-    "test:electron": "certa -r electron",
-    "pack": "npm pack"
+    "pack": "npm pack",
```

Comment:
Are test results included in the tarball? Please check and ensure that the packed tarball does not include test results. If they do - update paths, so that they are not included (they might need to be moved outside of `lib` directory).

Reply:
moved all tests results out of lib and into a new folder named test